typescript/cosmos-types/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@hyperlane-xyz/cosmos-types",
+  "version": "1.0.0",
+  "description": "Hyperlane TypeScript SDK types for the Cosmos Hyperlane SDK module",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "homepage": "https://www.hyperlane.xyz",
+  "repository": "https://github.com/hyperlane-xyz/hyperlane-monorepo",
+  "keywords": [
+    "hyperlane",
+    "cosmos",
+    "blockchain",
+    "sdk",
+    "types"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rm -rf ./dist && tsc",
```

Comment:
can we run the prettier after this? when i run `build`, it gives me a bunch of diffs that resolve after using `yarn prettier --fix`

Reply:
what are those diffs? I don't get any after running `yarn build` inside `typescript/cosmos-types`

Reply:
running prettier after `build` now